### PR TITLE
[GPU] Fix some bugs of cldnn eltwise kernel at bs_fs_yx_bsv32_fsv16 format.

### PR DESCRIPTION
**Pull request by jade-cho**
+ Add a condition so that the eltwise_simple_vload8 kernel is not selected when the tensor is not aligned as bsv32_fsv16 or bsv32_fsv32.
+ Optimize gws/lws of eltwise_ref kernel for bsv32_fsv16 format.

### CODE REVIEW

File: inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_vload8.cpp [hunk 1]
```diff
@@ -32,18 +32,27 @@ bool EltwiseKernel_vload8::Validate(const Params& params, const optional_params&
...
+                return false;
+        }
+        if ((ewParams.output.GetLayout() == DataLayout::b_fs_yx_fsv16 && ewParams.output.Feature().v % 16 != 0) ||
+            (ewParams.output.GetLayout() == DataLayout::b_fs_zyx_fsv16 && ewParams.output.Feature().v % 16 != 0) ||
+            (ewParams.output.GetLayout() == DataLayout::b_fs_yx_fsv4 && ewParams.output.Feature().v % 8 != 0) ||
+            ewParams.output.GetLayout() == DataLayout::fs_b_yx_fsv32 ||
+            (ewParams.output.GetLayout() == DataLayout::bs_fs_yx_bsv32_fsv16 &&
+                (ewParams.output.Feature().v % 16 != 0 || ewParams.output.Batch().v % 32 != 0)) ||
+            (ewParams.output.GetLayout() == DataLayout::bs_fs_yx_bsv32_fsv32 &&
+                (ewParams.output.Feature().v % 32 != 0 || ewParams.output.Batch().v % 32 != 0)))
```
Comment (isanghao) [inline inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_vload8.cpp:54]: Isn't it necessary to check `b_fs_yx_fsv32`?

Comment (jade-cho) [reply inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_vload8.cpp:54]: It is not that I added. Do you want to remove that b_fs_yx_fsv32 checking code?

Comment (jade-cho) [reply inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_vload8.cpp:56]: Added

Comment (yeonbok) [reply inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_vload8.cpp:56]: As I shared in offline discussion, actually we can allow this kernel for not alinged layouts either by 1) fix computing logic of tensor size or 2) fix dispatch data for work items to run for meaningful positions 

So I'd like to just ask : is this change decided by a performance results showing that using simd8 kernel was not that much better than using ref kernel? Or just for the work priority?

Comment (jade-cho) [reply inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_vload8.cpp:56]: For tensors of shape 32x3x512x512, the performance between the two kernels is similar. ref kernel also has accuracy issues including bugs, this PR should be included. Can't you optimize the vload8 kernel after merging this PR?

Comment (isanghao) [reply inference-engine/thirdparty/clDNN/kernel_selector/core/actual_kernels/eltwise/eltwise_kernel_vload8.cpp:56]: As mentioned offline, proper approach would be to optimize through fsv16 kernel, not vload8 kernel.

**Review by isanghao — APPROVED** [review]

**Pull request merged**